ui/perfherder/graphs/GraphTooltip.jsx

```diff
@@ -188,6 +193,11 @@ const GraphTooltip = ({
               &Delta; {displayNumber(deltaValue.toFixed(1))} (
               {(100 * deltaPercent).toFixed(1)}%)
             </p>
+            {isDatumAffected && (
+              <p className="small text-warning">
+                Could be affected by infra changes.
```

Comment:
This isn't that useful. We only show this message for the 1st data point next to a infra change.
There are many chances a user doesn't observe this message.

Wouldn't it make more sense to provide this message for multiple consecutive data points? That way, we increase the user's awareness.